app/api/slang.py

```diff
@@ -17,47 +34,84 @@ def generate_slang():
     return ret
 
 
-def generate_slang_internal():
-    """Implement internal call to generate and return a new slang word instead of API call."""
-    # Retrieve model from the session if it's already stored there.
+def _retrieve_info_from_session(model_type: str):
```

Comment:
```suggestion
def _retrieve_info_from_session(model_type: str) -> Tuple[nn.Module, Vocabulary, WordLevelDataset]:
```
I think a PyTorch model is an nn.Module strictly speaking, right. Or could we be more strict with the specific child class here, e.g. an RNNAnna. To err on the safe side it's probably best to stick with nn.Module